akka-actor/src/main/scala/akka/actor/RepointableActorRef.scala

```diff
@@ -236,26 +245,10 @@ private[akka] class UnstartedCell(val systemImpl: ActorSystemImpl,
     lock.lock // we cannot lose system messages, ever, and we cannot throw an Error from here as well
     try {
       val cell = self.underlying
-      if (cellIsReady(cell)) {
+      if (cellIsReady(cell))
         cell.sendSystemMessage(msg)
-      } else {
-        // systemMessages that are sent during replace need to jump to just after the last system message in the queue, so it's processed before other messages
-        val wasEnqueued = if ((self.lookup ne this) && (self.underlying eq this) && !queue.isEmpty()) {
```

Comment:
To make my question clear, I'm not looking for any more bug. I found the issue, I just wanted second review confirmation that I have not messed up anything. Thanks.